corehq/apps/accounting/tasks.py

```diff
@@ -351,6 +355,41 @@ def send_purchase_receipt(payment_record, core_product, domain,
     )
 
 
+@task(ignore_result=True)
```

Comment:
makes sense, done.  Might make sense to do this for more of the billing tasks as well.